numba/tests/test_array_manipulation.py

```diff
@@ -754,6 +762,74 @@ def _assert_raises(arr, val):
         val = np.array([-1e100])
         _assert_raises(arr, val)
 
+    def test_broadcast_to(self):
+        pyfunc = numpy_broadcast_to
+        cfunc = jit(nopython=True)(pyfunc)
+
+        # Tests taken from
+        # https://github.com/numpy/numpy/blob/75f852edf94a7293e7982ad516bee314d7187c2d/numpy/lib/tests/test_stride_tricks.py#L234-L257  # noqa: E501
+        data = [
+            [np.array(0), (0,)],
+            [np.array(0), (1,)],
+            [np.array(0), (3,)],
+            [np.ones(1), (1,)],
+            [np.ones(1), (2,)],
+            [np.ones(1), (1, 2, 3)],
+            [np.arange(3), (3,)],
+            [np.arange(3), (1, 3)],
+            [np.arange(3), (2, 3)],
+            # test if shape is not a tuple
+            [np.ones(0), 0],
+            [np.ones(1), 1],
+            [np.ones(1), 2],
+            # these cases with size 0 are strange, but they reproduce the behavior
+            # of broadcasting with ufuncs (see test_same_as_ufunc above)
```

Comment:
```suggestion
            # of broadcasting with ufuncs (see test_same_as_ufunc above)
```
is this comment relevant to Numba? (that test is in NumPy source?).

Reply:
Not relevant to numba